### ISSUE TITLE
strange result 

### ISSUE THREAD

**Issue by GarrickLin** [YuliangXiu/MobilePose-pytorch#6]
hi, I tried to train your code with few modification
- set resnet input size to 224x224
- set learning_rate to 1e-03 with adam optimizer

and the loss seems reasonable
```
[epoch 139] train loss: 11.97749722, valid loss: 302.35555648
[epoch 141] train loss: 12.36122017, valid loss: 300.01421413
[epoch 143] train loss: 12.04360196, valid loss: 297.55886086
[epoch 145] train loss: 11.31563594, valid loss: 300.16024249
[epoch 147] train loss: 10.84763370, valid loss: 291.31610589
[epoch 149] train loss: 10.44486987, valid loss: 290.63944685
[epoch 151] train loss: 11.03760799, valid loss: 300.89297566
[epoch 153] train loss: 10.26842959, valid loss: 293.75293954
[epoch 155] train loss: 9.67422970, valid loss: 288.11433823
[epoch 157] train loss: 9.24360814, valid loss: 296.10527976
[epoch 159] train loss: 9.42581674, valid loss: 296.30135754
==> checkpoint model saving to ./models/resnet/final-aug.t7
```
but when I tried to test, the result is strange
![mon_apr__9_17 25 07_2018_draw](https://user-images.githubusercontent.com/4505534/38491552-c1e8cd48-3c1e-11e8-8ebe-83d1eef9d66e.jpg)
![mon_apr__9_17 27 10_2018_draw](https://user-images.githubusercontent.com/4505534/38491554-c5dbd5da-3c1e-11e8-9228-9fdcca6677c3.jpg)
is this reasonable ?

**Comment by YuliangXiu** [comment]
The current version is pretty coarse and naive, it can only support **single full** body pose estimation, so you can stand a little far from the camera to make sure the full body appearing. 
I am working on a new version which can support both multi-person and be robust to the realistic scenes. Please wait for a few weeks.

**Comment by GarrickLin** [comment]
and this is the evaluation result, is this like yours ?
```
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.401
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets= 20 ] = 0.795
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets= 20 ] = 0.366
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.384
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.415
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.561
 Average Recall     (AR) @[ IoU=0.50      | area=   all | maxDets= 20 ] = 0.891
 Average Recall     (AR) @[ IoU=0.75      | area=   all | maxDets= 20 ] = 0.597
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.630
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.542
```

**Comment by YuliangXiu** [comment]
YES! It is very close to my results. If you can further improve this results, PR is welcome!

**Comment by GarrickLin** [comment]
Yes, I'm working on it. I'll tell you if there is any futher progress.

**Comment by YuliangXiu** [comment]
Very thanks.

**Comment by liuchanfeng165** [comment]
> YES! It is very close to my results. If you can further improve this results, PR is welcome!

不对，相差0.2左右啊，和原文提供的训练好的文件相比。怎么调的？我的训练了70+epoches，结果还是很低，loss下降很难
